esp/src/eclwatch/XrefFoundFilesWidget.js

```diff
@@ -70,6 +98,56 @@ define([
             return retVal;
         },
 
+        refreshActionState: function (event) {
+            var selection = this.grid.getSelected();
+            var hasSelection = selection.length;
+
+            registry.byId(this.id + "Delete").set("disabled", !hasSelection);
+            registry.byId(this.id + "Attach").set("disabled", !hasSelection);
+        },
+
+        _onDeleteFiles: function (event) {
+            var context = this;
+            var selections = this.grid.getSelected();
+            if (confirm(this.i18n.DeleteSelectedFiles)) {
+                var promises = [];
+                arrayUtil.forEach(selections, function (row, idx) {
+                    promises.push(WsDFUXref.DFUXRefArrayAction({
+                        request: {
+                            Cluster: context.params.Name,
+                            Type: 'Found',
+                            Action: 'Delete',
+                            XRefFiles_i2: row.Name
```

Comment:
Should probably be "XRefFiles_i0"?

In fact you can probably group all the actions into a single call using XRefFiles_i0, XRefFiles_i1, XRefFiles_i2...  (search for "flattenArray" for other places we do this.)